spec/migrations/20200331150436_rename_foreman_features_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_migration
+
+describe RenameForemanFeatures do
+  class MiqRolesFeature < ActiveRecord::Base; end
```

Comment:
This should not be needed. as you have the roles_feature_stub below, which is preferable.